### PR TITLE
Added image files

### PR DESCRIPTION
**Pull request by genjuro86**

### CODE REVIEW

File: Individual Assignments/Nam Young-Ju/Week 2/README.md [hunk 1]
```diff
@@ -6,6 +6,9 @@ Biography
 
 My name is Young-ju Nam, and I’m from South Korea. I am a first year MIDP student now. Before I came to the Georgetown University, I have been working at the Export-Import Bank of Korea for 10 years. I was in charge of providing financial support to overseas infrastructure projects such as power plants, water supply, hospitals, and railways to help Korean exporting companies expand their overseas business. To deepen my academic background and acquire rigorous quantitative analytical skills, I would devote myself to learn various programs during 2 years of MIDP. After graduation, I hope to be an ODA expert to make a better world.
```
Comment (alexacoon) [inline Individual Assignments/Nam Young-Ju/Week 2/README.md:7]: This is great! what does ODA stand for?

**Review by alexacoon — APPROVED** [review]
Fabulous!

File: Individual Assignments/Nam Young-Ju/Week 2/README.md (hunk 1 above)
Comment (genjuro86) [reply Individual Assignments/Nam Young-Ju/Week 2/README.md:7]: ODA stands for Official Development Assistance!! :) Thank you for a comment!

**Review by bnhippolyte — APPROVED** [review]
Looks good Nam

**Pull request merged**